integration_test.go

```diff
@@ -0,0 +1,129 @@
+package main_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"path/filepath"
+	"reflect"
+	"runtime"
+	"testing"
+
+	"github.com/atmiguel/cerealnotes/handlers"
+	"github.com/atmiguel/cerealnotes/models"
+	"github.com/atmiguel/cerealnotes/paths"
+	"github.com/atmiguel/cerealnotes/routers"
+)
+
+func TestLoginOrSignUpPage(t *testing.T) {
+	mockDb := &DiyMockDataStore{}
+	env := &handlers.Environment{mockDb}
+
+	handlers.SetEnvironment(env)
+	handlers.SetTokenSigningKey([]byte(""))
+
+	server := httptest.NewServer(routers.DefineRoutes())
+	defer server.Close()
+
+	resp, err := http.Get(server.URL)
```

Comment:
It's good practice to always `defer resp.Body.Close()` after making an http request
https://stackoverflow.com/questions/23928983/defer-body-close-after-receiving-response

```
If the returned error is nil, the Response will contain a non-nil Body which
the user is expected to close. If the Body is not closed, the Client's
underlying RoundTripper (typically Transport) may not be able to re-use a
persistent TCP connection to the server for a subsequent "keep-alive" request.
```

Reply:
Even if i don't use the respone.body?